indico/modules/events/layout/templates/layout.html

```diff
@@ -22,4 +19,59 @@
     {% call form_footer(form) %}
         <input class="i-button big highlight" type="submit" value="{% trans %}Save{% endtrans %}" data-disabled-until-change>
     {% endcall %}
+    <h2>{% trans %}Stylesheets{% endtrans %}</h2>
+    {{ form_header(css_form, classes='css-form', action=url_for('event_layout.upload_css', event)) }}
+        <div class="flashed-messages"></div>
+        {{ form_rows(css_form, skip_labels=true) }}
+    {% call form_footer(form) %}
+        <button class="i-button js-dropzone-upload highlight icon-upload" data-disabled-until-change>
+            {% trans %}Upload CSS file{% endtrans %}
+        </button>
+        <button class="i-button js-remove-button danger icon-remove{% if not css_form.css_file.data %} hidden{% endif %}"
+                data-href="{{ url_for('event_layout.delete_css', event) }}" data-method="DELETE">
+            {% trans %}Delete{% endtrans %}
+        </button>
+    {% endcall %}
+    <h2>{% trans %}Event Logo{% endtrans %}</h2>
+    {{ form_header(logo_form, classes='logo-form',  action=url_for('event_layout.upload_logo', event)) }}
+        <div class="flashed-messages"></div>
+        {{ form_rows(logo_form, skip_labels=true) }}
+    {% call form_footer(form) %}
+        <button class="i-button js-dropzone-upload highlight icon-upload" data-disabled-until-change>
+            {% trans %}Upload logo{% endtrans %}
+        </button>
+        <button class="i-button js-remove-button danger icon-remove{% if not logo_form.logo.data %} hidden{% endif %}"
+                data-href="{{ url_for('event_layout.delete_logo', event) }}" data-method="DELETE">
+            {% trans %}Delete{% endtrans %}
+        </button>
+    {% endcall %}
+    <script>
+        $(document).ready(function() {
+            $('.css-form, .logo-form').on('indico:fieldsSaved', function(e, response) {
+                var $form = $(this);
+                $form.find('.js-remove-button').toggleClass('hidden', !response.content)
```

Comment:
If you use the `weak-hidden` class you can use the normal `.toggle(..)` to show/hide instead of having to toggle the class